modules/virt-extended-selinux-policies-for-virt-launcher.adoc

```diff
@@ -6,18 +6,32 @@
 [id="virt-extended-selinux-policies-for-virt-launcher_{context}"]
 = Extended SELinux policies for virt-launcher pods
 
-The `container_t` SELinux policy for virt-launcher pods is extended with the following rules:
+The `container_t` SELinux policy for `virt-launcher` pods is extended to enable essential functions of {VirtProductName}.
 
-* `allow process self (tun_socket (relabelfrom relabelto attach_queue))`
-* `allow process sysfs_t (file (write))`
-* `allow process hugetlbfs_t (dir (add_name create write remove_name rmdir setattr))`
-* `allow process hugetlbfs_t (file (create unlink))`
+* The following policy is required for network multi-queue, which enables network performance to scale as the number of available vCPUs increases:
 
-These rules enable the following virtualization features:
+** `allow process self (tun_socket (relabelfrom relabelto attach_queue))` 
 
-* Relabel and attach queues to its own TUN sockets, which is required to support network multi-queue. Multi-queue enables network performance to scale as the number of available vCPUs increases.
+* The following policy allows `virt-launcher` to read files under the `/proc` directory, including `/proc/cpuinfo` and `/proc/uptime`:
 
-* Allows virt-launcher pods to write information to sysfs (`/sys`) files, which is required to enable Single Root I/O Virtualization (SR-IOV).
+** `allow process proc_type (file (getattr open read))`
 
-* Read/write `hugetlbfs` entries, which is required to support huge pages. Huge pages are a method of managing large amounts of memory by increasing the memory page size.
+* The following policy allows `libvirtd` to relay network-related debug messages, preventing excessive logging:
 
+** `allow process self (netlink_audit_socket (nlmsg_relay))`
+
+* The following policies allow `libvirtd` to access `hugetblfs`, which is required to support huge pages:
+
+** `allow process hugetlbfs_t (dir (add_name create write remove_name rmdir setattr))` 
+** `allow process hugetlbfs_t (file (create unlink))`
++
+[NOTE]
+====
+Huge pages are a method of managing large amounts of memory by increasing the memory page size.
```

Comment:
How about a pointer to the huge pages section instead?
(at `virt/virtual_machines/advanced_vm_management/virt-using-huge-pages-with-vms.html`)
This one line might be a bit of an over-simplification.